p2p-specs/p2p-interface.md

```diff
@@ -494,24 +495,38 @@ Request Content:
 
 ```
 (
-  mempool: Bytes32
-  offset: uint64
+  // cursor
+  Bytes32
```

Comment:
cursor: Bytes32

Reply:
This is meant to indicate this is now a value field, and not a container.